getting-started/bookshelf/app/app_test.go

```diff
@@ -28,16 +27,6 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func TestMainFunc(t *testing.T) {
-	wt := webtest.New(t, "localhost:8080")
-	m := testutil.BuildMain(t)
-	defer m.Cleanup()
-	m.Run(nil, func() {
```

Comment:
I wonder if we should still build the main package, but not necessarily run it (to make sure it compiles). What do you think?

Reply:
It all depends on how well exercised the app is.  It doesn't hurt, but I don't think it provides a great deal of additional utility.  I'll pull it back in.